Dockerfile

```diff
@@ -2,10 +2,15 @@ FROM node:18.16.0-alpine AS build
 
 WORKDIR /code
 
-ARG REACT_APP_BACKEND_URL
-ARG REACT_APP_DEBUG_MODE
-ARG REACT_APP_GITHUB_API_URL
-ARG REACT_APP_SEARCH_SOURCE
+ARG ARG_REACT_APP_BACKEND_URL
+ARG ARG_REACT_APP_DEBUG_MODE
+ARG ARG_REACT_APP_GITHUB_API_URL
+ARG ARG_REACT_APP_SEARCH_SOURCE
+
+ENV REACT_APP_BACKEND_URL=${ARG_REACT_APP_BACKEND_URL:-https://finesse.ninebasetwo.xyz/api}
```

Comment:
Thought we had discussed this being /api to ensure it goes to an upstream under the same domain?

Reply:
See https://github.com/ai-cfia/nachet-frontend/pull/85#issuecomment-1906661479